presto-tests/src/main/java/com/facebook/presto/tests/TestingPrestoClient.java

```diff
@@ -193,6 +193,9 @@ else if (type instanceof VarcharType) {
         else if (VARBINARY.equals(type)) {
             return value;
         }
+        else if (type.toString().equals("ObjectId")) { // So there's no circular dependency (presto-mongodb -> presto-tests -> presto-mongodb)
```

Comment:
Yeah, that would require a major refactoring of presto-tests.

In the short term, I would suggest adding an isolated test to the mongodb plugin test suite for that specific type that doesn't rely on presto-tests.